docker/service/src/main/java/uk/nhs/adaptors/scr/models/gpsummarymodels/EndTime.java

```diff
@@ -0,0 +1,10 @@
+package uk.nhs.adaptors.scr.models.gpsummarymodels;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class EndTime {
```

Comment:
Why create class just to store a string?

Reply:
For Mustache template to only populate this field when populated, if there is a better way to populate this let me know